docs/en/wallet-solution.rst

```diff
@@ -13,38 +13,80 @@ such as Credential Issuers and Relying Parties.
 
 These Credentials are a set of data that can uniquely identify a natural or legal person,
 along with other Qualified and non-qualified Electronic Attestations of Attributes,
-also known as QEAAs and EAAs respectively, or (Q)EAAs for short[1]. 
+also known as QEAAs and EAAs respectively, or (Q)EAAs for short [1]_. 
 
 Once a User installs the mobile app on their device, such an installation is referred to as a Wallet Instance for the User.
 
-By supporting the mobile app, the Wallet Provider enusers the security and reliability of the entire Wallet Solution,
+By supporting the mobile app, the Wallet Provider ensures the security and reliability of the entire Wallet Solution,
 as it is responsible for issuing the Wallet Attestation,
 which is a cryptographic proof about the authenticity and integrity of the Wallet Instance.
 
-Wallet Solution Requirements
+Requirements
```

Comment:
we need to have contextualized sections, "Requirements" alone would collid with other headings, in other chapters, named with the same name "Requirements".

```suggestion
Wallet Solution Requirements
```